sdk/aqueduct/integrations/salesforce_integration.py

```diff
@@ -87,11 +90,31 @@ def config(self, object: str) -> SaveConfig:
         Returns:
             SaveConfig object to use in TableArtifact.save()
         """
+        logger().warning(
```

Comment:
what's the difference between `print` and `logger.warning`?

Reply:
We should use logger() everywhere. It allows the user to configure the logging level (debug, info, warning, error, etc.) when creating the Aqueduct client.